From d30961916fcf18f8d4630dd501b1acbc29f102f2 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Mon, 27 Oct 2025 05:06:49 +0000
Subject: [PATCH 523/553] tests: build qrangemodel test only if Gui is
 available

The test has dependency to Qt::Gui.

Change-Id: I0dc0b113291439803f5968cee21b72330b76bf9e
Reviewed-by: Thiago Macieira <thiago.macieira@intel.com>
(cherry picked from commit 6c2ff0f9ca7e53e1f613f74630cc8ff99ffe7100)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 tests/auto/corelib/itemmodels/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/auto/corelib/itemmodels/CMakeLists.txt b/tests/auto/corelib/itemmodels/CMakeLists.txt
index 15c32b7a6f2..8304fa15bc6 100644
--- a/tests/auto/corelib/itemmodels/CMakeLists.txt
+++ b/tests/auto/corelib/itemmodels/CMakeLists.txt
@@ -2,8 +2,8 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 add_subdirectory(qstringlistmodel)
-add_subdirectory(qrangemodel)
 if(TARGET Qt::Gui)
+    add_subdirectory(qrangemodel)
     add_subdirectory(qabstractitemmodel)
     if(QT_FEATURE_proxymodel)
         add_subdirectory(qabstractproxymodel)
-- 
2.51.2

